test/transpiler/transpiler_double_test.ts

```diff
@@ -16,31 +16,814 @@ describe('Transpiler', () => {
                 .setSignature('double', WebAssemblyType.FLOAT_64)
                 .transpile('function double() { return 1.34; } ');
 
-            expect(wrapper.setFunctionName('double').call()).to.equal(1.34);
+            expect(wrapper.setFunctionName('double').call()).to.closeTo(1.34, 0.001);
```

Comment:
`to.equal()` also works here. Is there a reason why you used `closeTo()`?

Reply:
For consistency's sake and because imo you should never compare a double value without a delta, even if it is a non calculated value. But if you want it changed I wouldn't have a problem with that either.

Reply:
For consistency's sake and because imo you should never compare a double value without a delta, even if it is a constant value. But if you want me to change it, I wouldn't have a problem with that either.

Reply:
For consistency's sake and because imo you should never compare a double value without a delta, even if it is a constant value. But if you want it changed I wouldn't have a problem with that either.

Reply:
Remove it from the tests where it's not necessary.